[Delta] Nueva Estación 1/2

Spircen opened the pull request:
## What Does This PR Do
Cambia la estación base del diseño Cyberiad a Delta. Este cambio incluye multiples cambios al mapa y al sistema de radio control que cumple la estación, el nombre original del proyecto Kerberos es cambiado a Hispania a la par que se hacen los respectivos cambios en el mapa para que sea operativa, estos incluyen:

- Dormitorios con Bolt Doors
- Nano-Med Emergency alrededor de la estacion
- Shuttle de Ingenieria a satelite de la misma
- Agregamos placas base con nombre NSS Hispania 
- Reparamos areas respectivas de seguridad para que tengan acceso a parte de cada departamento
- Pod de Escape de ingenieria (Pod 4)
- Agregamos TSF Discovery a parte sur de la estacion
- Reemplazamos las salas de cirugías por los surgical cart cargados 
- Agregamos dormitorios a maint con el fin de evitar morir mientras te encuentras dentro de alguno. 
- Balanceamos el loot que se encuentra dentro de Maint
- Maquinas de ejercicio para el gimnasio en el area civil 
- Agregamos banderas departamentales a maint
- Cucarachas y Ratas a zonas de Maint
- Solo gente con acceso al puente de mando puede ir a zona de IA 
- Agregamos Transit animation a z2 de Delta para TSF
- Agregamos Ram y 2 hardsuits extras a la armeria
- Agregamos body scanner y NanoMed Plus a zonas de seguridad
- Agregamos spawn point a Brig Phyisician
- Remplazamos doors de librería de madera a doors normales 
- Agregamos peceras y kit para que el chef pueda tener peces
- Modificamos el archivo de carga de Delta por un typo
- Asigna de forma correcta multiples areas
- Reemplaza los lockers de armas de la nave del sindicato por hardsuit como debe ser
- Ampliamos duchas
- Eliminamos Autolathe Publico en Ciencias
- Posiciona de forma correcta Tachyon de Toxinas
- Agregamos Vending de Laptops
- Agregamos BODA a zona de maint
- Elimina mainboards APLU que vienen por default en robotica
- Agrega maquina dispensadora de robotista a su area
- Agrega respectivos cargo telepads a cargo y ciencias
- Agrega tres crates extras para balancear el starting founds que proporcionan estas
- Privacy Shutters CE, HoS, CmO, RD y QM
- Megafonos a CE, HoS, CmO, RD, QM, Capitan y Warden
- Consola de Despidos a QM
- Agregamos juego de garra a arcade y carpeta
- Agregamos infraestructura de NanoMob
- Agregamos blue and red gear para laser tag en arcade abandonada
- Piscina y sus respectivos animales
- Añade placa conmemorativa a arrivals y evac
- Añade equipo faltante a cocina candy maker, grill, cereal maker
- Añade vendedor de mascotas a cuarto de civiles
- Cambia los dispensadores de agua de botánica por los de larga duracion
- Añade Floodlight a botanica
- Añade Flasks al bar
- Añade Hispaniabox al Bar (Posicionada para solo escuchar dentro del bar)
- Añade monitor de mensajes a sala de server RnD (RD)
- Suelo reforzado Experimentor
- Locker de Parches y Pildoras faltante a medbay
- Emergency tanks vox y plasmaman medbay
- Hardsuits medicos almacen y suits de pandemias
- Bolsas de sangre a medbay
- Balancea los niveles de materiales respectivos para ciencias e ingenieria

## Why It's Good For The Game
Tenemos una nueva estación mas amplia y llena de nuevas posibilidades a la par que ofrece zonas mas optimas en base a los cambios de reglas y proporciona una nueva área de trabajo para todos los departamentos, esto sin duda alguna marca una separación parcial de paradise y ofrece a los jugadores una experiencias mucho mas distinta a lo que ya ofrecemos 

## Images of changes

                          Ingeniería (Privacy Shutters, Seguridad Departamental)
![image](https://user-images.githubusercontent.com/46639834/80583757-40b79080-89d6-11ea-89c5-8a67cdd875dd.png)

                          Ciencias (Privacy Shutters, Seguridad Departamental)
![image](https://user-images.githubusercontent.com/46639834/80583818-5a58d800-89d6-11ea-8089-97a35863f32a.png)

                           Robotica (Cambios, Placa de Evac y Consola en cuarto de servers)
![image](https://user-images.githubusercontent.com/46639834/80583886-78bed380-89d6-11ea-84eb-d75d34364715.png)

                          Medbay Lobby (Cambios, Seguridad Departamental)
![image](https://user-images.githubusercontent.com/46639834/80583917-870cef80-89d6-11ea-98e7-80e6d54468b3.png)

                       Medbay Lockers (Agregamos mas bolsas, huds, lockers y surgical cart)
![image](https://user-images.githubusercontent.com/46639834/80583983-a3a92780-89d6-11ea-9a94-4a827d781d88.png)

                       CmO Room y Almacen de ciencias (Hardsuits y trajes de pandemias)
![image](https://user-images.githubusercontent.com/46639834/80584069-c6d3d700-89d6-11ea-8a2e-a0b730dc2491.png)

                       Cargo (Seguridad Departamental y crates extras para Starting Founds)
![image](https://user-images.githubusercontent.com/46639834/80584137-de12c480-89d6-11ea-822d-c8f7cbe3c5d1.png)

                        Arcade de Maint (Crates no incluidas son de un test fallido)
![image](https://user-images.githubusercontent.com/46639834/80584230-fbe02980-89d6-11ea-9f10-40ff845dd455.png)

                                                          NSS HISPANIA
![image](https://user-images.githubusercontent.com/46639834/80658722-84e57800-8a4c-11ea-8815-73988ec73f07.png)

                        Armeria, Hardsuits Extras y Docking Port Gamma
![image](https://user-images.githubusercontent.com/46639834/80584337-203c0600-89d7-11ea-8516-99a3e4726c40.png)

                                          Articulos Extras Brig Physician
![image](https://user-images.githubusercontent.com/46639834/80584354-27631400-89d7-11ea-843a-d1e0b6a09a18.png)

                                                   Gimnasio
![image](https://user-images.githubusercontent.com/46639834/80658502-e48f5380-8a4b-11ea-84fd-74ab9abd9526.png)

## ¿Que viene en parte 2?
Reparare todo el sistema de disposals y de sensores para los emules, es demasiado extraño como funciona el sistema de disposals esto hace que sea enteramente funcional la estacion exceptuando estos dos detalles.

![_3fYL8i6Q-n-155t3dn_4jDknYN0aCPtudyMf63Csj0WcqbLRuyEIHKcG7ADvf27](https://user-images.githubusercontent.com/46639834/80657553-86617100-8a49-11ea-957e-c7c6af16bdca.gif)


## Changelog
:cl:
add: Cambios a Delta Station
add: Cambios a Z2 de Delta
/:cl:

Spircen committed:
Validacion Singularidad

Spircen committed:
Revert "Validacion Singularidad"

This reverts commit 6ba86a7b2a0b25a462c0335d2177b5999c71bbb8.

Spircen committed:
Merge remote-tracking branch 'upstream/master'

Spircen committed:
Merge remote-tracking branch 'upstream/master'

Spircen committed:
Merge remote-tracking branch 'upstream/master'

Spircen committed:
Merge remote-tracking branch 'upstream/master'

Spircen committed:
Merge remote-tracking branch 'upstream/master'

Spircen committed:
Merge remote-tracking branch 'upstream/master'

Spircen committed:
Merge remote-tracking branch 'upstream/master'

Spircen committed:
Merge remote-tracking branch 'upstream/master'

Spircen committed:
Merge remote-tracking branch 'upstream/master'

Spircen committed:
Merge branch 'master' into Delta

Spircen committed:
Merge remote-tracking branch 'upstream/master'

Spircen committed:
Merge branch 'master' into Delta

Spircen committed:
Delta Changes

Delta Dana Changes

Co-Authored-By: DanaDririon <danadririon@users.noreply.github.com>

Spircen committed:
Delta Base

Co-Authored-By: DanaDririon <danadririon@users.noreply.github.com>

Spircen committed:
Just Design Changes NO NAME CHANGES

Espero esto pase el travis, en teoria no es lo mismo pero es igual

Co-Authored-By: DanaDririon <danadririon@users.noreply.github.com>

Spircen committed:
Minor Typo

Spircen committed:
Mas cambios enormes a Delta

Spircen committed:
TSF Transit

Spircen committed:
Doors and delete some shit from space

Spircen committed:
Cargo Security and Armory Changes

Spircen committed:
Mas cambios varios y emergency nanomeds

Spircen committed:
More sciences and area changes

Spircen committed:
Muchos mas cambios

Spircen committed:
Lux Best Waifu

Evankhell561 commented:
![image](https://user-images.githubusercontent.com/49106866/80591425-ae21ec00-89eb-11ea-80cc-315d9c72db51.png)
Esta es un area de reunion en medio del pasillo, no creo que deba estar ahi. Deja las sillas si quieres pero pegalas más a las paredes.

Spircen commented:
Eso ni es mio, por mi lo borro. Ahora mismo estoy teniendo una revelacion y sube mi IQ mientras intento entender el sistema de disposals y por que no funciona.

Evankhell561 commented:
si no habrá nada más en esta mesa, separa un poco los objetos que estan amontonados la tile del sur.

![image](https://user-images.githubusercontent.com/49106866/80591713-25578000-89ec-11ea-9acd-ccdef3fdfc91.png)

Evankhell561 reviewed (changes requested):
![image](https://user-images.githubusercontent.com/49106866/80591906-82533600-89ec-11ea-8bbc-c0877c768baa.png)
veo una ablative armor en la armeria. Nosotros usamos un ablatibe technimierdas.

Evankhell561 reviewed (changes requested):
veo tiles moradas en med

Evankhell561 reviewed (changes requested):
![image](https://user-images.githubusercontent.com/49106866/80593563-7321b780-89ef-11ea-9060-4d1b1faea270.png)
Estos materiales eran para el autolathe que estaba frente a ciencias, a mi me gustaba el autolathe ahi porque sentia que toda la crew tenia acceso a el. Era algo que lo diferenciaba de hispania. Pero en fin, lo dejaras dentro de cargo esos materiales extra no tienen sentido, o reposicionalos dentro o no los añadas. También la cantidad de materiales dentro de ciencais deberia ser igual de hispania. la ultima vez que revisé no eran iguales en robotica (antes de este pr).

Spircen commented:
No sé por qué te desgastas si aún tengo el WIP, la de errores que aún hay que reparar y aún así andas de cherry pick en mi trabajo de casi 6 horas. 

STFU

Evankhell561 commented:
> No sé por qué te desgastas si aún tengo el WIP, la de errores que aún hay que reparar y aún así andas de cherry pick en mi trabajo de casi 6 horas.
> 
> STFU

Oh, te autoasignaste el wip, no lo vi. Normalmente lo ponemos en el titulo cuando hacemos un pr, tambien vi que habias dicho que solo pasarias ss cuando termines los cambios. Si aun esta en construccion adelante.

Evankhell561 commented on _maps/delta.dm:
```diff
@@ -16,16 +16,16 @@ Lovingly ported by Purpose2 to Paradise
-	#include "map_files\delta\delta.dmm"
-	#include "map_files\delta\z2.dmm"
+	#include "map_files\Delta\delta.dmm"
+	#include "map_files\Delta\z2.dmm"
```
deja el nombre del archivo en minusculas

Spircen replied:
No, eso no pasa travis.

Evankhell561 commented on _maps/delta.dm:
```diff
@@ -16,16 +16,16 @@ Lovingly ported by Purpose2 to Paradise
 
 	#define MAP_FILE "delta.dmm"
-	#define MAP_NAME "Kerberos"
+	#define MAP_NAME "Hispania"
```
El nombre se queda como Kerberos. Hispania es nuestro otro mapa

Spircen replied:
El map name se ocupa para el control de trafico y referencias a station name IC

Evankhell561 replied:
claro pero antes de esto decia "trafic blabla Kerberos", verdad?

Evankhell561 replied:
Lo que quiero decir es, este no es el NSS HISPANIA. Este es el NSS Kerberos. Quizá en el futuro haya que butizarlo distinto. But, de momento sigue siendo el NSS Kerberos.

Evankhell561 commented on code/modules/map_fluff/delta.dm:
```diff
@@ -1,9 +1,9 @@
 /datum/map/delta
-	name = "Delta"
-	full_name = "NSS Kerberos"
+	name = "Hispania"
```
lo mismo aca y en las lineas anteriores, hispania es nuestro otro mapa, no este.

Spircen replied:
Lo mismo de arriba

Spircen committed:
Cambios

Codigo de colores
Armeria Armor
Balance de minerales
"Sala de juntas fuera de sala de juntas"

Spircen commented:
Aquí están los cambios que habían pedido hace rato, ya que esta cosa pase y en PR que viene subo los cambios extras que pidan aquí que ya se esta sumando mucho en un solo PR 

![image](https://user-images.githubusercontent.com/46639834/80658669-5b2c5100-8a4c-11ea-9ca3-1eb34f879cc4.png)

Spircen commented:
Literalmente hay unas placas que dicen NSS Hispania por que ese es el nombre tanto del server como la estación.

Evankhell561 commented:
> Literalmente hay unas placas que dicen NSS Hispania por que ese es el nombre tanto del server como la estación.

Las placas las acabas de poner. Que nuestro servidor se llame asi no significa que ese nombre deba estar en todos nuestros mapas. El NSS HISPANIA ya existe y no es este. Es una segunda estacion funcional para nuestro servidor, eso sí, no significa que tenga que llamarse igual a la anterior.

Spircen commented:
Oh god entonces que quieres ¿Que quite todo y se llame kerberos? Yo quiero que se llame hispania y podamos hacer uso de las placas que tenemos.

Evankhell561 commented:
> Oh god entonces que quieres ¿Que quite todo y se llame kerberos? Yo quiero que se llame hispania y podamos hacer uso de las placas que tenemos.

Hacemos uso de las placas que tenemos en el NSS HISPANIA, este no es el NSS hispania. Es el segundo mapa de nuestra comunidad.

Spircen commented:
*dabea*
